hail/python/hail/ggplot/labels.py

```diff
@@ -72,3 +77,19 @@ def ylab(label):
         Label object to change the y-axis label.
     """
     return Labels(ylabel=label)
+
+
+def group_labs(**group_labels):
```

Comment:
Lets call this `labs` for consistency with ggplot.

Reply:
Updated!